src/assets/scss/base/_base.scss

```diff
@@ -9,37 +9,38 @@
 
 // Reset CSS
 * {
-	margin: 0;
-	padding: 0;
-	box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
 // Base Code
 html,
 body {
-	font-size: 100%;
+  font-size: 100%;
+  background-color: getColor("white.dark");
 }
 
 body {
-	-webkit-font-smoothing: antialiased;
-	color: getColor('black.base');
-	font-family: $font-primary;
-	font-size: $base__font-size;
-	text-rendering: optimizeLegibility;
-	line-height: $base__body__line;
+  -webkit-font-smoothing: antialiased;
+  color: getColor("black.light");
```

Comment:
body should take default value. Donot add any # color. because we need to reset every time when we set up the project.  If the color is need in Docs page then add it to `docs.scss` file